database/postgres/postgres_test.go

```diff
@@ -307,6 +307,49 @@ func TestPostgres_Lock(t *testing.T) {
 	})
 }
 
+func TestDrop(t *testing.T) {
```

Comment:
Rename to `TestMigrateDrop` and add a comment that mentions preventing a regression for https://github.com/golang-migrate/migrate/issues/164

It may be worth moving this test to `Test()` in `database/testing/testing.go` to test all DB driver implementations.

Reply:
Should I leave this test as-is? A new `TestMigrateDrop()` has been added to `database/testing/migrate_testing`, however this test makes sure that the Postgresql driver specifically is able to `Drop()` both version table and tables created through SQL, outside of migrations.

Reply:
I don't think this test is necessary anymore since all cases are covered by `dk.TestDrop()` and `dk.TestMigrateDrop()`

Reply:
Alright then.